beacon_node/store/src/iter.rs

```diff
@@ -15,15 +15,15 @@ impl<'a, T: EthSpec, U: Store> StateRootsIterator<'a, T, U> {
         Self {
             store,
             beacon_state: Cow::Borrowed(beacon_state),
-            slot: start_slot,
+            slot: start_slot + 1,
```

Comment:
I put the plus one first as the `next()` function pre-decrements `slot` before execution.
It was cleaner to add the plus one there to help dealing with case case when `slot == 0`
As when `start_slot = 0` we should return genesis block, but the pre-decrement in `next()`
would cause a negative overflow.